__tests__pet.test.js

```diff
@@ -0,0 +1,99 @@
+const Pet = require('./src/pet');
```

Comment:
This file could use a bit of a rejig. 
- Did you mean to create a folder called `__tests__` and put this file in there?
- Currently all the tests are wrapped in a single describe block, so it would appear at first glance that all the tests are related to the constructor (as you've called it on line 3)
Its good practice to separate out the tests by the function that uses them with separate describe blocks for each group